docs/conf.py

```diff
@@ -106,5 +106,5 @@
 simplepdf_vars = {
     "primary": "#ffcc00",
     "cover": "black",
-    "cover-bg": "url(pex-cover.png) no-repeat center",
+    "cover-bg": "url(pex-icon.png) no-repeat center",
```

Comment:
Sorry about that, https://jsirois.github.io/pex/_static/pex.pdf was missing the cover page artwork. Now re-deployed and looking correct.